index.html

```diff
@@ -5,330 +5,360 @@
-            </div>
-            <div class="holder">
-                <div id="lets" class="let">
-                    <p>LET'S CONNECT</p>
+            <div class="partone">
+                <div class="iamtext">
+                    <div class="primarytext">
+                        <h1> I’m Affax, <br> Glad to see you!
+                        </h1>
+                    </div>
+                    <div class="subtext">
+                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </p>
+                    </div>
+                    <div class="socialmedia">
+                        <p>LET'S CONNECT</p>
+                    </div>
+                    <div class="mediaicons">
+                        <ul>
+                            <li><img src="./Images/twitter.svg" alt="T"></li>
+                            <li><img src="./Images/linkedin.svg" alt="L"></li>
+                            <li><img src="./Images/medium.svg" alt="M"></li>
+                            <li><img src="./Images/github.svg" alt="G"></li>
+                            <li><img src="./Images/angellist.svg" alt="A"></li>
+                        </ul>
+                    </div>
                 </div>
-                <ul class="icon">
-                    <li class="first">
-                        <a href="first"><img src="./Images/twitter.svg" alt="twitter"></a>
-                    </li>
-                    <li class="second">
-                        <a href="second"><img src="./Images/Linkedin icon.svg" alt="Linkedin"></a>
-                    </li>
-                    <li class="third">
-                        <a href="third"><img src="./Images/github.svg" alt="github"></a>
-                    </li>
-                    <li class="fourth">
-                        <a href="fourth"><img src="./Images/angellist.svg" alt="angellist"></a>
-                    </li>
-                    <li class="fifth">
-                        <a href="fifth"><img src="./Images/medium.svg" alt="medium"></a>
-                    </li>
-                </ul>
+            </div>
             </div>
         </section>
-        <section class="section2">
-            <div class="container">
-                <div class="photo">
-                    <a href="#"> <img src="./Images/Snapshoot Portfolio.png" alt="SnapshootPortfolio"></a>
-                </div>
+        <section class=" listportfolio">
+            <div class="card1">
+                <div class="snapshot1"><img src="./Images/SnapshootPortfolio1w.png" alt="Photo"></div>
                 <div class="leftblock">
```

Comment:
- [ ] On the mobile screen, each project in the projects section is not horizontally aligned properly.
![image](https://user-images.githubusercontent.com/33767951/140235655-8d432a0e-156a-4d83-8221-f9ac5678b470.png)
- [ ] On the desktop screen, each project in the projects section is not horizontally aligned properly.
![image](https://user-images.githubusercontent.com/33767951/140235962-8e90545a-4a41-47a9-b39d-d076575a5706.png)